src/js/components/FileInput/FileInput.js

```diff
@@ -81,6 +81,7 @@ const FileInput = forwardRef(
     const [hover, setHover] = React.useState();
     const [dragOver, setDragOver] = React.useState();
     const aggregateThreshold = (multiple && multiple.aggregateThreshold) || 10;
+    const maxCount = multiple && multiple.max;
```

Comment:
Maybe it would be more aligned with prop name to just call this variable `max`

Reply:
Done